packages/ripple-binary-codec/test/hash.test.js

```diff
@@ -50,10 +50,8 @@ describe('Hash256', function () {
 })
 
 describe('Currency', function () {
-  test('Will throw an error for dodgy XRP ', function () {
-    expect(() =>
```

Comment:
Same here, this test should pass, because it's behavior we want.

Reply:
I disagree. The codec should not pass judgement. It is not middleware.